.github/workflows/release.yml

```diff
@@ -0,0 +1,159 @@
+          path: |
+            ${{ steps.srtool_build.outputs.wasm_compressed }}
+
+  publish-draft-release:
+    runs-on: ubuntu-latest
+    needs: ["build-runtimes"]
+    outputs:
+      release_url: ${{ steps.create-release.outputs.html_url }}
+      asset_upload_url: ${{ steps.create-release.outputs.upload_url }}
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Download srtool json output
+        uses: actions/download-artifact@v3
+
+      - name: Archive context output
+        uses: actions/upload-artifact@v3
+        with:
+          name: release-notes-context
+          path: |
+            **/*_srtool_output.json
+
+      - name: Add runtime info to changelog
+        run: |
+          >>CHANGELOG echo "## Runtime info"
+          for JSON in $(find . -name '*_srtool_output.json'); do
+            C() { >>CHANGELOG echo $@; }
+            WASM() { <${JSON} jq -r .runtimes.compact.subwasm.$1; }
+
+            C "### $(WASM core_version.specName)"
+            C '```'
+            C "🏋️ Runtime Size:           $(WASM size) bytes"
+            C "🔥 Core Version:           $(WASM core_version.specName)-$(WASM core_version.specVersion) $(WASM core_version.implName)-$(WASM core_version.implVersion).tx$(WASM core_version.transactionVersion).au$(WASM core_version.authoringVersion)"
+            C "🎁 Metadata version:       V$(WASM metadata_version)"
+            C "🗳️ system.setCode hash:    $(WASM proposal_hash)"
+            C "🗳️ authorizeUpgrade hash:  $(WASM parachain_authorize_upgrade_hash)"
+            C "🗳️ Blake2-256 hash:        $(WASM blake2_256)"
+            C "📦 IPFS:                   $(WASM ipfs_hash)"
+            C '```'
+          done
+      - name: Create draft release
+        id: create-release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Runtimes ${{ github.ref }}
+          body_path: CHANGELOG
+          draft: true
+
+  publish-runtimes:
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    needs: ["publish-draft-release"]
+    env:
+      RUNTIME_DIR: runtime
+    strategy:
+      matrix:
+        runtime: [ # this is the same matrix as above
+          # relay/para, runtime path, package name, chain
+          ["relay", "polkadot", "polkadot-runtime", "polkadot"],
+          ["relay", "kusama", "kusama-runtime", "kusama"],
+          #["relay", "parachains", "polkadot-runtime-parachains", "polkadot"], # ?
+          #["relay", "metrics", "polkadot-runtime-metrics", "polkadot"], # ?         
+          ["system-parachains", "collectives/collectives-polkadot", "collectives-polkadot-runtime", "collectives-polkadot"],
+          ["system-parachains", "asset-hubs/asset-hub-polkadot", "statemint-runtime", "statemint"],
+          ["system-parachains", "asset-hubs/asset-hub-kusama", "statemine-runtime", "statemine"],
```

Comment:
Should be called asset-hub-* and not statemint/statemine anymore.

Reply:
these come from the Cargo.toml of the runtimes, once this repo is synced to a more recent state this will be updated - until then this is required to build the current fellows runtimes revision.

Could you sync the main branch with the current parity revision? Then i can adjust this part.

Reply:
Actually, once that is done i'm going to replace this whole thing with a dynamic matrix generated from something like `find */runtimes -name "lib.rs" -not -path "*/common/*" -not -path "*/constants/*" | awk -F/ '{sub("/src/lib.rs",""); print "[\"" gensub("/" $NF "$", "", "g") "\",\"" $NF "\"]"}'` which should reduce maintenance burden by not hardcoding any of these things

Reply:
Ahh okay nice!